tests/src/org.ehcache/ehcache/3.10.8-jakarta/src/test/resources/META-INF/native-image/ehcache-test-metadata/reflect-config.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
- Only use `conditional` agent mode, cannot directly `nativeTest`. It took me almost three days to find out the specific functions of five specific classes from the `standard` agent mode, and they could not be traced by the graalvm agent in the `conditional` mode. 

- @chrisdennis Is there anything special about the design of the five classes? Or is it convenient to provide a more straightforward unit test? If we need to bring the content of this json, the PR cannot be merged.

Reply:
- This issue is resolved in the latest commit.